decidim-conferences/config/locales/en.yml

```
@@ -288,7 +288,23 @@ en:
           error: There was a problem updating a registration type for this conference.
           success: Conference registration type successfully updated.
       titles:
+        conference_admins: Conference admins
+        conference_invites: Invites
+        conference_partners: Partners
+        conference_registrations: Registrations
+        conference_speakers: Conference speakers
         conferences: Conferences
+        duplicate_conference: Duplicate conference
+        media_links: Media links
```

I know it's badly written, but I'd leave it as this for consistency: 
```suggestion
        media_links: Media Links
```

(Or change the other places where it says "Media Links" so it says "Media links", whatever solution you prefer as long as is consistent)